### PR TITLE
fix: [#184305952] review page fields spacing

### PR DESCRIPTION
**Pull request by somabadri**
## Description

Fixed spacing on review page fields. 

### Ticket

[184305952](https://www.pivotaltracker.com/story/show/184305952)

## Code author checklist

- [x] I have performed a self-review of my code
- [x] I have created and/or updated relevant documentation, if necessary
- [x] I have not used any relative imports
- [x] I have checked for and removed instances of unused config from CMS
- [x] I have pruned any instances of unused code
- [x] I have not added any markdown to labels, titles and button text in config

### CODE REVIEW

File: web/src/components/tasks/business-formation/review/section/ReviewLineItem.tsx [hunk 1]
```diff
@@ -18,19 +18,19 @@ export const ReviewLineItem = (props: Props): ReactElement => {
...
         {props.labelContextualInfo ? (
           <ContextualInfoButton text={`${props.label}:`} id={props.labelContextualInfo} />
         ) : (
           `${props.label}:`
         )}
       </div>
-      <Content>{props.formatter ? props.formatter(props.value) : props.value}</Content>
+      <div className={"grid-col flex-8"}>
+        <Content>{props.formatter ? props.formatter(props.value) : props.value}</Content>
+      </div>
```
Comment (jphechter) [inline web/src/components/tasks/business-formation/review/section/ReviewLineItem.tsx:33]: This is totally me nit picking, but is there a reason we weren't able to just provide these classNames to `Content`? It looks like they'd get appended to the className list for a wrapper div in that component.

**Review by jphechter — APPROVED** [review]
One question, but overall LGTM.

**Review by aloverso — APPROVED** [review]

File: web/src/components/tasks/business-formation/review/ReviewStep.test.tsx [hunk 2]
```diff
@@ -177,8 +177,20 @@ describe("Formation - ReviewStep", () => {
...
   });
 
+  it("does not display incorporators label within review step when incorporators are empty", async () => {
+    await renderStep({ legalStructureId: "c-corporation" }, { signers: [] });
+    expect(
+      screen.queryByText(markdownToText(Config.formation.fields.incorporators.label))
+    ).not.toBeInTheDocument();
+  });
+
+  it("does not display signers label within review step when singers they are empty", async () => {
```
[the remark this answers is not in the document]
Comment (somabadri) [reply web/src/components/tasks/business-formation/review/ReviewStep.test.tsx:187]: yes!

File: web/src/components/tasks/business-formation/review/section/ReviewLineItem.tsx (hunk 1 above)
Comment (somabadri) [reply web/src/components/tasks/business-formation/review/section/ReviewLineItem.tsx:33]: good catch, we can just pass into content, making the change.

Comment (somabadri) [reply web/src/components/tasks/business-formation/review/section/ReviewLineItem.tsx:33]: good catch, we can just pass into `Content`, making the change.

**Pull request merged**